applications/Bright_Disputes.md

```diff
@@ -0,0 +1,180 @@
+
+Voteing phase will start when all Juries and Judge confirm their assignment for the dispute, by providing an escrow. If someone will not confirm their assigment, Owner / Contractor can trigger smart contract to repick persons from the Juries pool. 
+
+In the voting phase, the Jury votes for one of the parties. The vote is encrypted with the public key of the Judge and a proof of a valid vote is delivered to smart contract. From now only the Judge can read the votes.
+
+At the end of the voting phase, the Judge is responsible for counting all the votes and issuing a verdict. If there is no majority of the votes (at least 60% against one party), verdict can not be unequivocal. In that case the dispute round need to be repeated, but the number of juries is increated by two. What is worth to mention is that votes are not revealed, so Juries doesn't know how others vote. Thats why Judge need to provides a proof (zk-snark) of his correct counting of the votes. When there is a majority of votes, the Judge issues a verdict and make votes public. Now Judge and the Juries which were in the majority of the votes are rewarded by their work. Juries whose votes are in the minority are penalized, by slashing their escrows. If Judge or Jury will not fulfill his duties (take action in time), their will lose their escrows.
+
+Beneath is the diagram showing the basic logic of the Bright Disputes, which were here described.
+
+![](https://i.imgur.com/jWWw54D.png)
+
+### Project Details
+
+
+
+
+
+
+"Bright Disputes" is a smart contract fully written in ink! From the development perspective, the most chalenging and interesting part will be the zk-snarks. Privacy is very important in our dispute system while the Juries publishes the encrypted votes, and the Judge counts them to reach a verdict. Role of the Judge requires honesty, he is the only one who can decrypt the votes. That's why we want to use zk-snarks to prove Judge honesty and correctness of the vote counting. Especially when the verdict is not clear and we need to repeat the voting round. The jury cannot see how others vote, otherwise the verdict will not be fair.
+
+Based on our knowlage of zk-snarks ([zk-snarks-in-substrate](https://brightinventions.pl/blog/zk-snarks-in-substrate-part-1/)), using this technology have some consequences. Creating of zk-snarks will generate some artifacts (verification / proving key). We will provide a CLI, which will simplify this process. As it shown on the diagram below Juries / Judge are going to use CLI to communicate with the smart contrant, while Owner / Contractor can use polkadot.js dApp.
+
+![](https://i.imgur.com/WjZczZx.png)
+
+Our target plaform for the deployment will be AlephZero, which provides zk-snarks verification.
+
+This project will not provide any UI. All the functionality can be tested by the polkadot.js dApp.
+
+## Team :busts_in_silhouette:
+
+### Team members
+
+- Michał Graliński - Rust Developer
+- Katarzyna Łukasiewicz - Project Manager
```

Comment:
Is a project manager really needed for a self-contained project with a single dev?